File: 2_Kernel/JavaKernel/src/main/java/com/seagosoft/tws/data/PositionMultiData.java
package com.seagosoft.tws.data;

import com.ib.client.Contract;
import com.ib.client.Decimal;

public class PositionMultiData {

    public int reqId;
    public String account;
    public String modelCode;
    public Contract contract;
    public Decimal pos;
    public double avgCost;


    public PositionMultiData(int reqId, String account, String modelCode,
                             Contract contract, Decimal pos, double avgCost) {
        this.reqId = reqId;
        this.account = account;
        this.modelCode = modelCode;
        this.contract = contract;
        this.pos = pos;
        this.avgCost = avgCost;
    }
}

File: 3_OrderSystem/KotlinEWrapperBackend/src/main/kotlin/com/seagosoft/tws/wrapper/structs/PositionMultiData.kt
package com.seagosoft.tws.wrapper.structs

import com.ib.client.Contract
import com.ib.client.Decimal


class PositionMultiData(
    var reqId: Int, var account: String, var modelCode: String,
    contract: Contract, var pos: Decimal, var avgCost: Double
) {
    var contract: Contract = contract
}